src/libguestfish.sh

```diff
@@ -2,6 +2,8 @@
 # A major assumption here is that the disk image uses OSTree
 # and also has `boot` and `root` filesystem labels.
 
+# We don't want to use libvirt for this, it inhibits debugging
+export LIBGUESTFS_BACKEND=direct
```

Comment:
:+1: I think in one of my commits for the unpriv work I just put this in cmdlib.sh or somewhere so that it was set everywhere. Might rework where all of these global vars get set at some point.